Doc/library/pickle.rst

```diff
@@ -511,6 +511,30 @@ The :mod:`pickle` module exports three classes, :class:`Pickler`,
       Release the underlying buffer exposed by the PickleBuffer object.
 
 
+.. _pickle-cli:
+
+Command-line interface
+----------------------
```

Comment:
Usually we put the CLI docs near the end of the page, after the library usage. Shall we move this down?

Reply:
It was looks like right place, right after `Module interface` and everything below seems like additional information
table of content:
![Screenshot from 2025-03-13 17-25-24](https://github.com/user-attachments/assets/312f5485-d5be-44bb-87e1-4eb34ffc5321)

Reply:
Although this additional information seems most relevant to library use?

Reply:
so between which sections do you think the place is more suitable?

Reply:
After "Examples".

Reply:
moved, but before `see also` section, right?